<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="encrypt_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrypt_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_textbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>751</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="key_textbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>741</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="input_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="output_textbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>751</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="output_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QLabel" name="key_err_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>741</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>error</string>
    </property>
   </widget>
   <widget class="QLabel" name="input_err_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>741</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>error</string>
    </property>
   </widget>
   <widget class="QComboBox" name="input_comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Text</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hex</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="key_size_label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0 / 0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
